File: app/src/main/java/com/luizroseiro/testeandroidv2/models/StatementData.java
package com.luizroseiro.testeandroidv2.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class StatementData {

    @SerializedName("title")
    @Expose
    private String title;

    @SerializedName("desc")
    @Expose
    private String desc;

    @SerializedName("date")
    @Expose
    private String date;

    @SerializedName("value")
    @Expose
    private double value;

    public String getTitle() {
        return title;
    }

    public String getDesc() {
        return desc;
    }

    public String getDate() {
        return date;
    }

    public String getFormattedDate() {
        String[] dateValues = date.split("-");
        return dateValues[2] + "/" + dateValues[1] + "/" + dateValues[2];
    }

    public double getValue() {
        return value;
    }

}

File: app/src/main/java/pt/felipegouveia/bankapp/data/statements/model/StatementData.kt
package pt.felipegouveia.bankapp.data.statements.model

import com.google.gson.annotations.SerializedName
import java.util.*

data class StatementData(
    @SerializedName("title")
    val title: String? = null,
    @SerializedName("desc")
    val desc: String? = null,
    @SerializedName("date")
    val date: Date? = null,
    @SerializedName("value")
    val value: Double = 0.0
)